handler/proxy.go

```diff
@@ -117,10 +117,7 @@ func (p *Proxy) RoundTrip(req *http.Request) (*http.Response, error) {
 	}
 
 	transport.RemoveConnectionHeaders(beresp.Header)
-
-	for _, h := range transport.HopHeaders {
-		beresp.Header.Del(h)
-	}
+	transport.RemoveHopHeaders(beresp.Header)
```

Comment:
See 
```
	for _, h := range transport.HopHeaders {
		req.Header.Del(h)
	}
```
in line 96.
If this is changed, too, `transport.HopHeaders` can become `transport.hopHeaders`

Reply:
I would leave it as it is since we do not support Trailer headers atm.